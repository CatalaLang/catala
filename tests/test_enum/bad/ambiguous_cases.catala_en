## Article

```catala
declaration enumeration E:
  -- Case1

declaration enumeration F:
  -- Case1

declaration scope A:
  context e content E

scope A:
  definition e equals Case1
```
