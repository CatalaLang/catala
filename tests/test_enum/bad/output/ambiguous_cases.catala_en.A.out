[ERROR] This constructor name is ambiguous, it can belong to E or F. Desambiguate it by prefixing it with the enum name.
[ERROR] 
[ERROR]   --> test_enum/bad/ambiguous_cases.catala_en
[ERROR]    | 
[ERROR] 14 |   definition e equals Case1
[ERROR]    |                       ^^^^^
[ERROR]    + Article
